data/rulesets.ts

```diff
@@ -530,58 +530,26 @@ export const Rulesets: {[k: string]: FormatData} = {
-						.replace(/(Greninja|Gourgeist|Pumpkaboo|Xerneas|Silvally|Urshifu|Dudunsparce)(-[a-zA-Z?-]+)?/g, '$1-*')
-						.replace(/(Zacian|Zamazenta)(?!-Crowned)/g, '$1-*'); // Hacked-in Crowned formes will be revealed
-					this.add('poke', pokemon.side.id, details, teraTypeNote);
-				}
+			this.add('clearpoke');
+			for (const pokemon of this.getAllPokemon()) {
+				const teraTypeNote = (this.ruleTable.has(`teratypepreview`) && pokemon.teraType) ?
+					`[teraType] ${pokemon.teraType}` : ``;
+				const details = pokemon.details.replace(', shiny', '')
+					.replace(/(Greninja|Gourgeist|Pumpkaboo|Xerneas|Silvally|Urshifu|Dudunsparce)(-[a-zA-Z?-]+)?/g, '$1-*')
+					.replace(/(Zacian|Zamazenta)(?!-Crowned)/g, '$1-*'); // Hacked-in Crowned formes will be revealed
+				this.add('poke', pokemon.side.id, details, teraTypeNote);
 			}
-			if (this.showOpenTeamSheets) {
-				let buf = 'raw|';
+			this.makeRequest('teampreview');
+			if (this.ruleTable.has(`teratypepreview`)) {
 				for (const side of this.sides) {
-					buf += Utils.html`<div class="infobox" style="margin-top:5px"><details><summary>Open Team Sheet for ${side.name}</summary>${Teams.export(side.team, {hideStats: true})}</details></div>`;
-				}
-				if (this.ruleTable.has('broadcastopenteamsheets')) {
-					this.add(buf);
-					this.add('clearpoke');
-				} else {
-					for (const side of this.sides) {
-						this.addSplit(side.id, [buf]);
-						this.addSplit(side.id, ['clearpoke']);
-					}
-				}
-				for (const pokemon of this.getAllPokemon()) {
-					const moveList = pokemon.getMoveList();
-					let setNotes = `[name] ${pokemon.name}`;
-					let details = pokemon.details;
-					if (pokemon.item) setNotes += `|[item] ${pokemon.item}`;
-					if (pokemon.ability) setNotes += `|[ability] ${pokemon.ability}`;
-					if (pokemon.teraType) setNotes += `|[teraType] ${pokemon.teraType}`;
-					if (pokemon.species.id === 'zacian' && pokemon.item === 'rustedsword') {
-						const ironHead = moveList.indexOf('ironhead');
-						if (ironHead >= 0) moveList[ironHead] = 'behemothblade';
-						details = details.replace('Zacian', 'Zacian-Crowned');
-					} else if (pokemon.species.id === 'zamazenta' && pokemon.item === 'rustedshield') {
-						const ironHead = moveList.indexOf('ironhead');
-						if (ironHead >= 0) moveList[ironHead] = 'behemothbash';
-						details = details.replace('Zamazenta', 'Zamazenta-Crowned');
-					}
-					setNotes += `|[moveList] ${moveList.join(';')}`;
-					if (this.ruleTable.has('broadcastopenteamsheets')) {
-						this.add('poke', pokemon.side.id, details, setNotes);
-					} else {
-						const message = ['poke', pokemon.side.id, details, setNotes].join('|');
-						for (const side of this.sides) this.addSplit(side.id, [message]);
+					let buf = ``;
+					for (const pokemon of side.pokemon) {
+						buf += buf ? ` / ` : `raw|${side.name}'s Tera Types:<br />`;
+						buf += `<psicon pokemon="${pokemon.species.id}" /><psicon type="${pokemon.teraType}" />`;
```

Comment:
This is nice!

Reply:
For the record, this is not from my code (it was introduced last January).

Reply:
Oh, you should probably rebase in that case.

Reply:
From what I can tell, based on the result ("Files Changed" tab), the subject `teraType` -gallery code is  *not* considered part of my changes.
Also, I was under the impression that modifying this PR (name/title is inaccurate) such as by rebasing, renaming, etc. would cause issues
(... and I have mishandled these PRs enough already.)

(Sorry if this is terribly erroneous, I am still not especially familiar with Git /GitHub.)

Reply:
> Also, I was under the impression that modifying this PR (name/title is inaccurate) such as by rebasing, renaming, etc. would cause issues

You would normally be playing very safe but following such guidelines, but you have already messed with the PR by doing [Merge 'origin/master' into openTeamSheets](https://github.com/smogon/pokemon-showdown/pull/9605/commits/e58bec3f4f333cea18f094839605d5b04d12747c)

That's why you see below the error: <code>"This branch cannot be rebased due to conflicts"</code>. Since you are new with Git, resolving conflicts might be challenging, so you could also do a backup and work from a fresh PR from an up-to-date repository. You may also ask for help in #development and/or #techcode.